portal/client/debug.log

```diff
@@ -0,0 +1 @@
+[0118/111306.894:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
lets remove this file and add the line in gitignore to not to include these `*.log` files